Jint.Tests/Runtime/UuidTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Jint.Tests.Runtime.Domain;
+using System;
+using Xunit;
+
+namespace Jint.Tests.Runtime
+{
+    public class UuidTests : IDisposable
+    {
+        private readonly Engine _engine;
+
+        public UuidTests()
+        {
+            _engine = new Engine(o => o.AddObjectConverter(new UuidConverter()))
```

Comment:
What do you think about adding an extension method on Options named `AddUuid()` which would register the converter and also call CreateUuidConstructor ?

Reply:
Sounds good. Would need some API refactoring.

Reply:
Can we get away with AddObjectConverter<T>? not sure if we can bind the constructor with that though.

Reply:
@viceice I think we would need a way to add Action<Engine> to a list that would be executed by the engine when it has initialized the options object.

Reply:
What about to add `Engine.ObjectConverters`  as ` public List<IObjectConverter> ObjectConverters => Options._ObjectConverters`?  so we can add `IObjectConverter` after `Engine` contructor finished?

Then the `AddUuid()` extension would be very easy.

Reply:
I prefer not to since we are looking at ways to reuse engines. Having all the configuration kept in options would allow us to reuse it on any engine instance once cleared of any state. We could even add a list of scripts. Or use these options in a factory object for pooling.

Instead of calling these methods from Engine, we can also have a method Apply(engine) such that the engine code would not change every time we add options. And a Options.Configure (Action<Engine>) to register custom logic. I can add this to your PR if you prefer.

Reply:
Ok, feel free to change the PR. 🙃

Reply:
Maybe I need to give you write access to my fork, because it's in an org.

Reply:
If you haven't unallowed me, then by default I can write on the branch you used to submit a PR